phpunit.xml

```diff
@@ -68,6 +68,9 @@
     <testsuite name="eZ\Publish\API\Repository\Values\Content">
       <directory>eZ/Publish/API/Repository/Tests/Values/Content</directory>
     </testsuite>
+    <testsuite name="eZ\Publish\Core\Repository\Strategy\ContentThumbnail">
+      <directory>eZ/Publish/Core/Repository/Strategy/ContentThumbnail/Tests</directory>
```

Comment:
Please move the tests to `eZ/Publish/Core/Repository/Tests` and drop this change. There are too many separate test sources already.